components/molecule/photoUploader/src/ThumbCard.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import SUIContext from '@s-ui/react-context'
+import cx from 'classnames'
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
+
+import {BASE_CLASS_NAME} from './config'
+const THUMB_CARD_CLASS_NAME = `${BASE_CLASS_NAME}-thumbCard`
+const IMAGE_THUMB_CARD_CLASS_NAME = `${THUMB_CARD_CLASS_NAME}-image`
+const ICON_THUMB_CARD_CLASS_NAME = `${THUMB_CARD_CLASS_NAME}-iconContainer`
+
+const ThumbCard = ({
+  callbackDeleteItem,
+  callbackRetryUpload,
+  callbackRotateItem,
+  deleteIcon,
+  index,
+  image,
+  mainPhotoLabel,
+  rejectPhotosIcon,
+  retryIcon,
+  rotateIcon
+}) => {
+  const hasErrors = image.hasErrors
+
+  const counterClass = cx(`${THUMB_CARD_CLASS_NAME}-counter`, {
+    [`${THUMB_CARD_CLASS_NAME}-mainCounter`]: index === 0
+  })
+
+  return (
+    <div className="sui-MoleculePhotoUploader-thumbCard">
+      <div className={counterClass}>
+        {index === 0 ? mainPhotoLabel : index + 1}
+      </div>
+      <div className="sui-MoleculePhotoUploader-thumbCard-imageContainer">
+        {hasErrors ? (
+          <div className={`${ICON_THUMB_CARD_CLASS_NAME}`}>
+            <AtomIcon size={ATOM_ICON_SIZES.extraLarge}>
+              {rejectPhotosIcon()}
+            </AtomIcon>
+          </div>
+        ) : (
+          <img src={image.preview} className={IMAGE_THUMB_CARD_CLASS_NAME} />
+        )}
+      </div>
+      <div className="sui-MoleculePhotoUploader-thumbCard-actions">
+        <div
+          className="sui-MoleculePhotoUploader-thumbCard-button"
+          onClick={e => callbackDeleteItem(index)}
```

Comment:
if you don't use the event just don't take it: `onClick={() => callbackDeleteItem(index)}`